env/production/ecs/terragrunt.hcl

```diff
@@ -41,18 +45,31 @@ dependency "s3" {
 }
 
 inputs = {
-  subnet_id                 = dependency.network.outputs.private_subnet_id
-  sg_id                     = dependency.network.outputs.csv_etl_sg_id
-  unmasked_metrics_s3_arn   = dependency.s3.outputs.unmasked_metrics_arn
-  masked_metrics_s3_arn     = dependency.s3.outputs.masked_metrics_arn
-  masked_metrics_bucket     = dependency.s3.outputs.masked_metrics_id
-  unmasked_metrics_bucket   = dependency.s3.outputs.unmasked_metrics_id
-  csv_etl_repository_url    = dependency.ecr.outputs.create_csv_repository_url
-  create_csv_repository_arn = dependency.ecr.outputs.create_csv_repository_arn
-  billing_tag_key           = "CostCentre"
-  billing_tag_value         = "CovidShield"
-  masked_image_tag          = "4819f93a1a68f0147be3536513cee4e07d766f47"
-  unmasked_image_tag        = "4819f93a1a68f0147be3536513cee4e07d766f47"
+  subnet_id                           = dependency.network.outputs.private_subnet_id
+  sg_id                               = dependency.network.outputs.csv_etl_sg_id
+  unmasked_metrics_s3_arn             = dependency.s3.outputs.unmasked_metrics_arn
+  masked_metrics_s3_arn               = dependency.s3.outputs.masked_metrics_arn
+  masked_metrics_bucket               = dependency.s3.outputs.masked_metrics_id
+  unmasked_metrics_bucket             = dependency.s3.outputs.unmasked_metrics_id
+  csv_etl_repository_url              = dependency.ecr.outputs.create_csv_repository_url
+  create_csv_repository_arn           = dependency.ecr.outputs.create_csv_repository_arn
+  server_metrics_etl_repository_url   = dependency.ecr.outputs.server_metrics_etl_repository_url
+  server_metrics_etl_repository_arn   = dependency.ecr.outputs.server_metrics_etl_repository_arn
+  appstore_metrics_etl_repository_url = dependency.ecr.outputs.appstore_metrics_etl_repository_url
+  appstore_metrics_etl_repository_arn = dependency.ecr.outputs.appstore_metrics_etl_repository_arn
+  billing_tag_key                     = "CostCentre"
+  billing_tag_value                   = "CovidShield"
+  masked_image_tag                    = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  unmasked_image_tag                  = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  masked_server_tag                   = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  unmasked_server_tag                 = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  masked_appstore_tag                 = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  unmasked_appstore_tag               = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  cpu_units                           = 512
+  memory                              = 1024
```

Comment:
Since this is going prod, want to bump this to `4096` or `8192`?

Reply:
yes should be 8192
also might want to upgrade cpu units as well

Reply:
makes sense for cpu bump as well